samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/CalculatedQuestionAnswerIfc.java

```diff
@@ -56,4 +56,6 @@ public interface CalculatedQuestionAnswerIfc {
      */
     public String getMatch();
 
+    public boolean isFromui();
```

Comment:
I'm fine with this method, but I'd like a more descriptive name that clarifies what is returning.